.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 .DS_Store
 node_modules
+.idea/
```

Comment:
this is unrelated, just hiding the .idea directory for Intellij IDE tools (like Pycharm/webstorm/intellij/etc)